bindata/network/ovn-kubernetes/common/alert-rules.yaml

```
@@ -18,11 +18,36 @@ spec:
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/NodeWithoutOVNKubeNodePodRunning.md
         description: |
           Networking is degraded on nodes that do not have a functioning ovnkube-node pod. Existing workloads on the
-          node may continue to have connectivity but any additional workloads will not be provisioned on the node. Any
-          network policy changes will not be implemented on existing workloads on the node.
+          node may continue to have connectivity but any changes to the networking control plane will not be implemented.
       expr: |
         (kube_node_info unless on(node) (kube_pod_info{namespace="openshift-ovn-kubernetes",pod=~"ovnkube-node.*"}
         or kube_node_labels{label_kubernetes_io_os="windows"})) > 0
       for: 20m
       labels:
         severity: warning
+    - alert: OVNKubernetesNodePodAddError
+      annotations:
+        summary: OVN Kubernetes is experiencing pod creation errors at an elevated rate.
+        description: OVN Kubernetes experiences pod creation errors at an elevated rate for the last 5 minutes. The
+          pods will be retried.
+      expr: |
+        (sum(rate(ovnkube_node_cni_request_duration_seconds_bucket{command="ADD",err="true"}[5m]))
+          /
+        sum(rate(ovnkube_node_cni_request_duration_seconds_bucket{command="ADD"}[5m])))
```

I believe you want `ovnkube_node_cni_request_duration_seconds_count` rather than `ovnkube_node_cni_request_duration_seconds_bucket`? The latter is for computing percentiles.

```suggestion
        (sum(rate(ovnkube_node_cni_request_duration_seconds_count{command="ADD",err="true"}[5m]))
          /
        sum(rate(ovnkube_node_cni_request_duration_seconds_count{command="ADD"}[5m])))
```

---

it might also be useful to sum by instance/node maybe?

```suggestion
        (sum by(instance) (rate(ovnkube_node_cni_request_duration_seconds_bucket{command="ADD",err="true"}[5m]))
          /
        sum by(instance) (rate(ovnkube_node_cni_request_duration_seconds_bucket{command="ADD"}[5m])))
```